addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -18,8 +18,25 @@
                     <a class="o_wslides_fs_toggle_sidebar d-flex align-items-center px-3" href="#" title="Lessons">
                         <i class="fa fa-bars"/><span class="d-none d-md-inline-block ml-1">Lessons</span>
                     </a>
-                    <a class="o_wslides_fs_review d-flex align-items-center px-3" t-att-href="slide.channel_id.website_url + '?active_tab=review'" title="Reviews" t-if="slide.channel_id.allow_comment">
-                        <i class="fa fa-pencil"/><span class="d-none d-md-inline-block ml-1">Write a review</span>
+                    <a class="o_wslides_fs_review d-flex align-items-center" title="Reviews" t-if="slide.channel_id.allow_comment">
+                        <t t-call="portal_rating.rating_stars_static_popup_composer">
+                            <t t-set="rating_avg" t-value="rating_avg"/>
+                            <t t-set="rating_total" t-value="rating_count"/>
+                            <t t-set="object" t-value="channel"/>
+                            <t t-set="token" t-value="channel.access_token"/>
+                            <t t-set="hash" t-value="message_post_hash"/>
+                            <t t-set="pid" t-value="message_post_pid"/>
+                            <t t-set="default_message" t-value="last_message"/>
+                            <t t-set="default_message_id" t-value="last_message_id"/>
+                            <t t-set="default_rating_value" t-value="last_rating_value"/>
+                            <t t-set="default_attachment_ids" t-value="last_message_attachment_ids"/>
+                            <t t-set="force_submit_url" t-value="'/slides/mail/update_comment' if last_message_id else False"/>
+                            <t t-set="disable_composer" t-value="not channel.can_review"/>
+                            <t t-set="_link_btn_classes" t-value="'d-inline-block text-white font-weight-light shadow-none'"/>
+                            <t t-set="icon" t-value="'fa fa-pencil'"/>
+                            <t t-set="text_classes" t-value="'d-none d-md-inline-block'"/>
```

Comment:
Not sure why '_link_btn_classes' starts with an underscore but then would be good to have text_classes start with an underscore as well for consistency.